### PR TITLE
cleanup: resolves gofumpt linter issues

### PR DESCRIPTION
**Pull request by yati1998**
This commit runs the command
"gofumpt -l -w ." to resolve the gofumpt
linter issues. Gofumpt is a modified fork
of gofmt. It enforces a stricter format than gofmt.

Updates: #1586

Signed-off-by: Yati Padia <ypadia@redhat.com>

### CODE REVIEW

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
Remove vendor changes we don't need this one?

File: e2e/rbd.go [hunk 1]
```diff
@@ -1945,10 +1944,9 @@ var _ = Describe("RBD", func() {
 						}
 					}()
 
-					var pvc = &v1.PersistentVolumeClaim{}
-					pvc, err = loadPVC(pvcPath)
-					if err != nil {
-						e2elog.Failf("failed to load PVC with error %v", err)
+					pvc, pvcErr := loadPVC(pvcPath)
```
Comment (nixpanic) [inline e2e/rbd.go:1947]: I'm surprised the change of naming `err` is suggested. It seems this is the only location where that is done?

**Review by nixpanic — CHANGES_REQUESTED** [review]
no concerns, except for the `vendor/` directory changes.

It might be simpler to split this PR into multiple smaller ones in order to reduce the number of conflicts it might cause. Consider doing that in case you notice it needs several manual rebases.

File: e2e/rbd.go (hunk 1 above)
Comment (yati1998) [reply e2e/rbd.go:1947]: No No. Actually the static-check was failing for this code, as the `err` here was overlapping the one declared above.

**Pull request closed without merging**